src/components/tabs/tab.js

```diff
@@ -1,12 +1,12 @@
 import React, { useCallback } from "react"
 import { StyledTab } from "./styled"
 
-export const Tab = ({ label, index, onChange, ...rest }) => {
+export const Tab = ({ index, onChange, ...rest }) => {
   const onClick = useCallback(() => onChange && onChange(index || 0), [index, onChange])
 
   return (
     <StyledTab onClick={rest.disabled ? undefined : onClick} {...rest}>
-      {label}
+      {rest.label}
```

Comment:
out of curiosity, this change will pass `label` prop to StyledTab, is this intentional?

Reply:
Yes, this change was intentional. [For reference](https://github.com/netdata/cloud-frontend/pull/2681/files#diff-052cc627f52854365d9eba7cdbb3ab0fd1a9bb845e3d1b8c328c37467c0cfac9)

Reply:
` <Tab label={<TabLabel label="linux" />} as={StyledButton}>`
Button takes it's label as a prop.

Reply:
@jjtsou thanks!